### ISSUE TITLE
Meaning of the dataset

### ISSUE THREAD

**Issue by sybhdfw** [TeCSAR-UNCC/Pishgu#2]
Hello, I am interested in your research, can you provide a copy of the code for processing the dataset? Or explain the meaning of the following fields after the dataset processing, thanks

**Comment by arkayareddy** [comment]
Hi @sybhdfw  What are the fields you are referring to ?

**Comment by sybhdfw** [comment]
> Hi @sybhdfw What are the fields you are referring to ?

Hi, @arkayareddy , what I'm asking about is the trajectory prediction field, I noticed that the author uses the NGSIM dataset but doesn't give a procedure for processing the data, so I have a little problem with the dataset field.

**Comment by vinit-katariya** [comment]
Hello @sybhdfw ,

The data preprocessing is done using the method described in https://arxiv.org/pdf/1805.06771.pdf. Please find the preprocessing code on their repo (https://github.com/nachiket92/conv-social-pooling).

**Comment by sybhdfw** [comment]
> Hello @sybhdfw ,
> 
> The data preprocessing is done using the method described in https://arxiv.org/pdf/1805.06771.pdf. Please find the preprocessing code on their repo (https://github.com/nachiket92/conv-social-pooling).

Hi, I followed the link in the README to download the processed dataset, and I found that the NGSIM dataset is some txt file, while the CS-LSTM processed data is mat file, and they are not the same. In addition, I found that the training set and test set in the program are the same during reproduction, can you provide a copy of the complete program for the data processing?

**Comment by sybhdfw** [comment]
Hi  @vinit-katariya ，
I followed the link in the README to download the processed dataset, and I found that the NGSIM dataset is some txt file, while the CS-LSTM processed data is mat file, and they are not the same. In addition, I found that the training set and test set in the program are the same during reproduction, can you provide a copy of the complete program for the data processing?

**Comment by vinit-katariya** [comment]
@sybhdfw ,

Thank you for pointing that out. Please find the updated datasets [here ](https://drive.google.com/file/d/16xKlIgvZQrpi0Wm6sPpKyhGjQRPIwFW0/view?usp=sharing)and the ngsim.pt file in the [model folder](https://github.com/TeCSAR-UNCC/Pishgu/tree/main/models/NGSIM).

**Comment by vinit-katariya** [comment]
Closing because there was no further issues.

**Comment by sybhdfw** [comment]
Hi, I have a problem reproducing the NGSIM results of your program, I processed the raw data using the data preprocessing code of CS-LSTM as you said, but the result after processing is not the same as the dataset you provided, can you provide a copy of the data preprocessing code? Also, I found that your error on the NGSIM dataset is very small, did you add filtering and other operations in the data preprocessing process?
